src/Ombi/ClientApp/app/landingpage/landingpage.component.ts

```diff
@@ -41,4 +49,22 @@ export class LandingPageComponent implements OnInit {
             this.mediaServerStatus = x;
         });
     }
+
+    public ngOnDestroy() {
+        clearInterval(this.timer);
+    }
+
+    public cycleBackground() {
+        setTimeout(() => {
```

Comment:
So the `this.timer` calls `this.cycleBackground` every 10 seconds, and then we have another two timers in here?

Once that clears the background and another that sets it. Why do we have this?

Reply:
Previously I was running into sync issues therefore I added the timeout timers. With the new improvements I made they are redundant, will quickly push a new commit with the timers removed.